index.html

```diff
@@ -276,5 +276,25 @@ <h3 class="drp_text">Skills</h3>
     </div>
 
   </section>
+  <section class="contactMe">
+      <h1 class="contactTxt">Contact me</h1>
+      <p 
+        class="Contact_body_text">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it 
+      </p>
+      <form class="	form" action="https://formspree.io/f/xnqrezql" method="post" >
```

Comment:
- [ ] Nice job integrating the form with the Formspree service. :+1: Let's work on some minor details to achieve the project requirements. 
- Kindly put placeholder text in your inputs to let visitors know what to put there. ✍️
- Please add some more vertical padding on your inputs to match the design. 🧐️
- Currently, the submit button is not aligned relative to the inputs. Kindly align the button to the left. 😎️

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/17243325/220691710-00329960-10fa-4543-84a4-2d009c0020cb.png" width="300">|<img src="https://user-images.githubusercontent.com/17243325/220691727-d603f8cd-b5a2-4eea-afd6-a9e4d5aaf1ca.png" width="300">|